readme.md

```diff
@@ -1,6 +1,6 @@
 # Hibernate OGM
 
-Version: 5.0.0.Final - 13 May 2016
+Version: 5.1.0-SNAPSHOT
```

Comment:
We usually only update the date/version during releases. Fine with me, just let's not forget to re-add the date during the next release.